### DISCUSSION TITLE
Deploy PHP7.4 application

### DISCUSSION THREAD

**Discussion by robinwo** [dokku/dokku#7149] [Q&A]
### Description of problem

I tried deploying an app still running PHP7.4. Unfortunately the Heroku buildpack dropped support, requiring PHP8 minimum.

Very much understand why upgrading is important, but while I’m working on that, I’m unable to get my current website up and running again.

Any help on how I might work around this, and still be able to deploy a PHP7.4 app?

### dokku report $APP_NAME

-

### Additional information
I did try using a versioned tag of the Heroku buildpack (#188-200) but that doesn’t work, still gives the minimum required PHP version is not available.

### Output of failing deploy after running: dokku trace:off

```shell
-
```

### Output of failing deploy after running: dokku trace:on

```shell
-
```

**Comment by josegonzalez** [comment]
I converted this to a discussion as it's not really a bug or anything we can fix in Dokku.

You could use a Dockerfile instead of the buildpack, or use the `pack` builder with packet buildpacks and try their PHP buildpack. I think the nixpacks builder might also work well for your use-case.

**Comment by robinwo** [comment]
Will have a look, thanks! Also this shows the importance of not fully relying on third party buildpacks. Good to be aware of!
